src/main/java/com/synopsys/integration/alert/common/descriptor/config/context/ChannelDistributionDescriptorActionApi.java

```diff
@@ -140,7 +140,8 @@ private DescriptorActionApi getProviderActionApi(final FieldModel fieldModel) {
                                                                          .filter(providerDescriptor -> providerDescriptor.getName().equals(providerName))
                                                                          .findFirst();
         if (foundProviderDescriptor.isPresent()) {
```

Comment:
This can be replaced by: 
```
return foundProviderDescriptor.map(api -> api.getActionApi(ConfigContextEnum.DISTRIBUTION)).orElse(null);
```
I would also be in favor of not returning null when we have an Optional; since this method is private, it doesn't matter as much, so it's up to your discretion.

Reply:
I would also be in favor of returning an optional over a null even if it is a private method